test/controllers/color_controller_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Elixirlight.ColorControllerTest do
+  use Elixirlight.ConnCase
+
+  defstruct [:color]
+
+  test "GET /api/healthcheck", %{conn: conn} do
+    conn = get conn, "/api/healthcheck"
+    assert json_response(conn, 200)
+  end
+
+  test "GET /api/color", %{conn: conn} do
+    color_as_json = Poison.encode!(%{color: "#ff0000"})
+    conn = get conn, "/api/color"
+    assert json_response(conn, 200) =~ color_as_json
```

Comment:
I don't think you can do this. I think you need two assertions here.
```
assert json_response(conn, 200)
assert conn.response_body =~ color_as_json
```

`json_response` doesn't return anything so the =~ is being called on `Kernel` it seems. https://hexdocs.pm/phoenix/Phoenix.ConnTest.html#json_response/2